notes/meetings/2021-12/december-15.md

```diff
@@ -0,0 +1,45 @@
+# December 15, 2021 - Meeting Notes
+
+**Faciliators:**
+
+| Name | Abbreviation |
+| - | - | - |
+| Dominik Wilkowski | DW |
+| Esther Semo | ES |
+| Elise Chant | EC |
+
+## Resources
+
+- [Survey Summary Results](assets/survey-results.pdf)
+- [Survey Full Results](assets/survey-results-full.csv)
+
+## Prior
+
+* Maintainers have read proposed RFCs
+
+## Survey findings
+
+* ES to show results
+
+## Prioritisation of actions related to survey findings
+
+* TODO 
+
+## Creating a skills register
+
+* TODO 
+
+## Maintainers commitments
+
+* Min time spend
+* TODO
+
+## RFC approvals
+
+* [Remove pancake](https://github.com/designsystemau/RFCs/pull/1)
+* [Simplify monorepo (DRAFT)](https://github.com/designsystemau/RFCs/pull/2)
+* [Separate framework entry points (DRAFT)](https://github.com/designsystemau/RFCs/pull/3)
+* [Docs site improvements](https://github.com/designsystemau/RFCs/pull/4)
+* [Create STANDARDISED-ACCESSIBILITY-TESTING.md](https://github.com/designsystemau/RFCs/pull/6)
+* [Update Grid 12](https://github.com/designsystemau/RFCs/pull/11)
+* [Distribute unminified and unprefixed CSS](https://github.com/designsystemau/RFCs/pull/16)
```

Comment:
Last part:

```suggestion
* [Distribute unminified and unprefixed CSS](https://github.com/designsystemau/RFCs/pull/16)

## Assigning people for implementation

 * [Remove pancake](https://github.com/designsystemau/RFCs/pull/1)
* [Simplify monorepo (DRAFT)](https://github.com/designsystemau/RFCs/pull/2)
* [Separate framework entry points (DRAFT)](https://github.com/designsystemau/RFCs/pull/3)
* [Docs site improvements](https://github.com/designsystemau/RFCs/pull/4)
* [Create STANDARDISED-ACCESSIBILITY-TESTING.md](https://github.com/designsystemau/RFCs/pull/6)
* [Update Grid 12](https://github.com/designsystemau/RFCs/pull/11)
* [Distribute unminified and unprefixed CSS](https://github.com/designsystemau/RFCs/pull/16)
```

Reply:
Re "## Assigning people for implementation", above comment you also say "Implementation can then be assigned to a few people between those meetings.", so which is better?